Makefile

```diff
@@ -2,8 +2,9 @@ IMAGE_REGISTRY ?= quay.io
 IMAGE_REPO ?= nmstate
 
 HANDLER_IMAGE_NAME ?= kubernetes-nmstate-handler
-HANDLER_IMAGE_TAG ?= latest
-HANDLER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(HANDLER_IMAGE_NAME):$(HANDLER_IMAGE_TAG)
+#HANDLER_IMAGE_SUFFIX ?= :latest
```

Comment:
why do we keep this and why the latest has `:`?

Reply:
Reading around, looks like tagging with latest is not good practices, you have to just do no tag and push and you can still get latest, that's why this HANDLER_IMAGE_SUFFIX is added here.

But the commented one is trash, let me remove it.